src/components/Collaborative/Components/LinkToolBar/FloatingLinkToolbar.tsx

```diff
@@ -35,8 +35,12 @@ const FloatingLinkToolbar = () => {
         clickEdit();
     }, [clickEdit]);
 
-    const handleInputChange = (event: ChangeEvent<HTMLInputElement>) => {
-        setHref(event.target.value);
+    const handleInputChange = ({
+        target: { value },
+    }: ChangeEvent<HTMLInputElement>) => {
+        // Prevent href with double http protocol
+        const href = value.replace(/(https?:\/\/)(https?:\/\/)+/, "$1");
```

Comment:
suggestion: create a regex variable outside of this function and make it global to this file. If you do like this the regex will be compiled every time the function is called, if you put on a variable and make it global it will compiled only once during the build.